settings/development.py

```diff
@@ -77,6 +78,10 @@
                'https://stats.g.doubleclick.net',
                'https://www.facebook.com',
                'https://www.googleadservices.com',
+               'http://organize-analytics.herokuapp.com',
+               'https://ams-login.dotomi.com/',
```

Comment:
It is related to some image sources that were missing. When I tried to view the images, nothing appeared in the browser. So I am removing them.